### ISSUE TITLE
ddx broken

### ISSUE THREAD

**Issue by strauss-s** [perlancar/perl-Data-Dump-Color#13]
Hi. I recently updated my system and now Data::Dump::Color::ddx seems to be broken:
```
#!/usr/bin/env perl

use strict;
use warnings;
use utf8;
use Data::Dump::Color;

Data::Dump::Color::dd( {text=>"Hello"});
Data::Dump::Color::dump( {text=>"Hello"});
Data::Dump::Color::ddx( {text=>"Hello"});
```

Result:

```
{ text => "Hello" }
{ text => "Hello" }
Can't call method "get_item_color" on an undefined value at /usr/local/share/perl/5.32.1/Data/Dump/Color.pm line 68.
```

Versions:
```
pmvers Data::Dump::Color
0.248
```
```
perl -v

This is perl 5, version 32, subversion 1 (v5.32.1) built for x86_64-linux-gnu-thread-multi
(with 47 registered patches, see perl -V for more detail)
```

It looks like ``$ct_obj`` does not get a value, because ``sub dump`` is not called.

**Comment by scottchiefbaker** [comment]
FWIW I can recreate this issue (with the above example) on my Perl installation as well.

**Comment by scottchiefbaker** [comment]
Any updates on this? I'd love to see this get fixed

**Comment by scottchiefbaker** [comment]
FYI: Super easy to recreate this:

    perl -E 'use Data::Dump::Color; ddx(9)'

**Comment by perlancar** [comment]
Sorry, I saw the GitHub notification but didn't have time to look at it until now. Fixed in 0.249 (commit ea00562).
